untraced/ykcompile/src/stack_builder.rs

```diff
@@ -13,26 +13,26 @@ use std::convert::{TryFrom, TryInto};
 #[derive(Default, Debug)]
 pub(super) struct StackBuilder {
     /// Keeps track of how many bytes have been allocated.
-    stack_top: u64,
+    stack_top: usize,
```

Comment:
I'm surprised this works without additional `try_from` changes elsewhere. I thought there was some DWARF-related nonsense, but maybe we evolved that out already..

Reply:
I was pleasantly surprised too!